package.json

```diff
@@ -25,11 +25,11 @@
     "package-maven-build-scan-capture-extension": "mvn clean package -f maven-build-scan-capture-extension/pom.xml",
     "package-build-scan-setup-maven-main": "ncc build build-scan-setup-maven/src/main.ts --out build-scan-setup-maven/dist/main",
     "package-build-scan-setup-maven-post": "ncc build build-scan-setup-maven/src/post.ts --out build-scan-setup-maven/dist/post",
-    "package-build-scan-setup-maven": "npm run package-build-scan-setup-maven-main && npm run package-build-scan-setup-maven-post && npm run package-maven-build-scan-capture-extension",
+    "package-build-scan-setup-maven": "npm-run-all --parallel package-build-scan-setup-maven-main package-build-scan-setup-maven-post package-maven-build-scan-capture-extension",
     "package-build-scan-setup": "npm run package-build-scan-setup-maven",
```

Comment:
I'm not sure why you have extra steps of redirection like this. I'd remove this until it proves necessary/useful.

Reply:
This is a left over from the Gradle/Maven cohabitation. It is not [yet] required indeed and I removed it 👍